syllabus/8-XCM/2-XCVM/XCVM_slides.md

```diff
@@ -143,37 +328,175 @@ Notes:
+
+TODO: Move elsewhere.
+Native tokens are referenced by the location to their system.
+
+---v
+
+### USDT from Asset Hub
+
+`PalletInstance(50)/GeneralIndex(1984)`
+
+<diagram class="mermaid limit size-70">
+graph TD
+    Polkadot(Polkadot):::disabled-->AssetHub("📍 Asset Hub (1000)")
+    Polkadot-->BridgeHub("Bridge Hub (1002)"):::disabled
+    AssetHub-->Alice(Alice):::disabled
+    AssetHub-->AssetsPallet(Pallet Assets)
+    AssetsPallet-->Asset(USDT)
+    linkStyle 0 opacity:0.3
+    linkStyle 1 opacity:0.3
+    linkStyle 2 opacity:0.3
+    classDef disabled opacity:0.3
+</diagram>
+
 ---v
 
-## More complete XCVM operation
+### USDT from Bridge Hub
+
+`../Parachain(1000)/PalletInstance(50)/GeneralIndex(1984)`
+
+<diagram class="mermaid limit size-70">
+graph TD
+    Polkadot(Polkadot)-->AssetHub("Asset Hub (1000)")
+    Polkadot-->BridgeHub("📍 Bridge Hub (1002)")
+    AssetHub-->Alice(Alice):::disabled
+    AssetHub-->AssetsPallet(Pallet Assets)
+    AssetsPallet-->Asset(USDT)
+    BridgeHub-->Polkadot
+    linkStyle 1 opacity:0.3
+    linkStyle 2 opacity:0.3
+    linkStyle 5 stroke-dasharray:5
+    classDef disabled opacity:0.3
+</diagram>
+
+---v
+
+### Reanchoring to the rescue
 
 <diagram class="mermaid">
 graph LR
-    subgraph Program
-        WithdrawAsset-->BuyExecution
-        BuyExecution-->DepositAsset
-        DepositAsset
+    subgraph OutgoingMessage[Outgoing message from Bridge Hub]
+        USDTBridgeHub(USDT from Bridge Hub's perspective)
     end
-    Program-.->Fetch
-    Fetch(Fetch Instruction)-->Execute(Execute instruction)
-    Execute-->Fetch
-    Execute-.->Registers
-    subgraph Registers
-        Holding(Holding)
-        Origin(Origin)
-        ErrorRegister(Error)
-        ErrorHandler(Error Handler)
-        AppendixRegister(Appendix)
-        More(...)
+    USDTBridgeHub--Reanchoring-->USDTAssetHub
+    subgraph IncomingMessage[Incoming message in Asset Hub]
+        USDTAssetHub(USDT from Asset Hub's perspective)
+    end
+</diagram>
+
+---
+
+## 🤹 Cross-consensus transfers
+
+Notes:
+
+The two ways of transferring assets between consensus systems are teleports and reserve transfers.
+
+---v
+
+### 1. Asset teleportation
+
+<img rounded style="width: 500px;" src="../../../assets/img/8-XCM/teleport.png" />
+
+Notes:
+
+Teleportation works by burning the assets on the source chain and minting them on the destination chain.
+This method is the simplest one, but requires a lot of trust, since failure to burn or mint on either side will affect the total issuance.
+
+---v
+
+### 1.1. Example: System parachains?
+
+<diagram class="mermaid">
+graph LR
+    BridgeHub(Bridge Hub)--"Trust"-->AssetHub(Asset Hub)
+</diagram>
+
+---v
+
+### 1.2. Example: Polkadot and Kusama?
+
+<diagram class="mermaid">
+graph LR
+    Polkadot(Polkadot)--"No trust"-->Kusama(Kusama)
+</diagram>
+
+---v
+
+### 2. Reserve asset transfers
+
+<img rounded style="width: 400px;" src="../../../assets/img/8-XCM/reserve-tx.png" />
```

Comment:
Please move the images inside the 8-XCM directory, and pick them from there (delete the old ones).
After that move the above should be:
```suggestion
<img rounded style="width: 500px;" src="../img/8-XCM/reserve-tx.png" />
```